EpikCord/__init__.py

```diff
@@ -2305,6 +2305,56 @@ def latency(self):
     def average_latency(self):
         return sum(self.latencies) / len(self.latencies)
 
+    def add_task(self, task:Callable[...,None], interval: Optional[int] = None, **kwargs):
+        """Adds a background task (Tasks that run silently in the background)
+
+        This task can be used to monitor the bot's performance, change and send stuff routinely
+        through webhooks, etc
+        You can set the time of when it starts, stops and  add which arguments to give to the task
+        You may also define whether to run once or run indefinitely
+
+        Args:
+            task (Callable): The function to run on the background
+            interval (Optional[int]): The interval (in seconds) for a delay between when the instance of the task stops and starts again .Defaults to 5.
+            instances (Optional[int]): The number of instances to start after delay. Usually unlimited
+            start (Optional[float]): The time to start, you may use Client.utils.get_time_since_epoch() to give the seconds since python epoch to this arg
+            until (optional[float]): The time to end, you may use Client.utils.get_time_since_epoch() to give the seconds since python epoch to this arg
+
+
+        """
+        kwargs["interval"] = interval
+        async def full_task(task, **kwargs):
+            async def task_func(interval):
+                try:
+                    await task()
+                except Exception as e:
+                    raise TaskFailedError(f"Task failed due to: {e}")
+
+                await asyncio.sleep(interval)
+
+            interval = int(kwargs.get("interval", 5))
```

Comment:
could have a better support for passing `h`, `m`, `s`, ...
See dpy implementation

Reply:
Will be implementing support for h,m,s tmrw, I am not well

Reply:
tmrw? more like `d` for days?
Could be nicer to use plain keywords tho...

Reply:
tmrw = Tomorrow
He meant to say he will do it tomorrow.